boards/arm64/fvp_base_revc_2xaemv8a/fvp_base_revc_2xaemv8a_smp_ns_defconfig

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022 Arm Limited (or its affiliates). All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_FVP_AEMV8A=y
+CONFIG_SOC_FVP_BASE_REVC_2XAEMV8A=y
+CONFIG_BOARD_FVP_BASE_REVC_2XAEMV8A=y
+
+CONFIG_XIP=n
+
+CONFIG_THREAD_STACK_INFO=y
+
+# Enable Timer and Sys clock
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_ARM_ARCH_TIMER=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+# Enable serial port
+CONFIG_UART_PL011=y
```

Comment:
Would you like to remove some CONFIGs that are not needed already, like `CONFIG_UART_PL011`?

Reply:
Yep